tensorflow/core/framework/queue_interface.h

```diff
@@ -78,7 +78,7 @@ class QueueInterface : public ResourceBase {
                      DoneCallback callback) = 0;
 
   // Returns true if a given queue is closed and false if it is open.
-  virtual bool closed() = 0;
+  virtual const bool closed() = 0;
```

Comment:
Should be

    virtual bool closed() const = 0;

Making return values const doesn't do anything in C++.